test/unit/mongodb/mongodb-group-registry-test.js

```diff
@@ -59,6 +59,7 @@ var iotAgentLib = require('../../../lib/fiware-iotagent-lib'),
     },
     mongo = require('mongodb').MongoClient,
     mongoUtils = require('./mongoDBUtils'),
+// This test will be removed if at the end the /iot/services API (now deprecated) is removed
```

Comment:
This is not the right point to include this mark. Please have a look to https://github.com/telefonicaid/iotagent-node-lib/pull/780/files#r440636381

(Same for all the other usages of this line in the current PR)

Reply:
I have added the new tests and updated the old ones. Fixed by https://github.com/telefonicaid/iotagent-node-lib/pull/780/commits/dacb188a03c3c0063c88e9eb488d6a9f95825ff5